src/EventStore.Client/EventStoreGrpcClient.Read.cs

```diff
@@ -16,15 +15,17 @@ public partial class EventStoreClient {
 		/// <param name="direction">The direction in which to read. <see cref="Direction"/></param>
 		/// <param name="position">The position to start reading from.</param>
 		/// <param name="maxCount">The maximum count to read.</param>
+		/// <param name="operationOptions"><see cref="EventStoreClientOperationOptions" /> to perform the operation with.</param>
```

Comment:
Given that these methods are now private, should the documentation be moved to the public ones?